package.json

```diff
@@ -26,6 +26,7 @@
     "lint:changeset": "changeset status",
     "lint:dependency-version-consistency": "check-dependency-version-consistency .",
     "lint:eslint": "turbo run --continue lint:eslint",
+    "lint:knip": "knip",
```

Comment:
I guess we can run `knip` as part of `ci.yml` (where we run Prettier, ESLint etc.). Curious to see how long it takes to check the whole repo!

Reply:
@kachkaev Is there a reason we have various workflows bundled in the general `ci.yml` file, but not in all cases?

I see linting and tests inside it, but we also have some tests outside (static analysis, at least... namely Semgrep and CodeQL).

Reply:
Semgrep and CodeQL checks are created using third-party YAML templates. If our plan is to run them as frequently as Linting / Tests, they can become jobs inside `ci.yml`. I guess it’s is a topic for another discussion.

Because `knip` is configured as a `package.json` dependency (just as `eslint` or, say, `check-dependency-version-consistency`), we can run it together with other local linting tools.